Library/Homebrew/requirements/maximum_macos_requirement.rb

```diff
@@ -12,7 +12,11 @@ def initialize(tags)
 
   def message
     <<-EOS.undent
-      OS X #{@version.pretty_name} or older is required.
+      This formula either does not compile or function as expected on newer
+      OS X versions than #{@version.pretty_name} due to an upstream incompatibility.
+
+      You can try to install the formula with `--HEAD` but one may not be
+      defined or the issue may still be present upstream.
```

Comment:
Happy to if you'd prefer not to keep it. I added it because we deliberately don't MMOSR `head`s in the impacted formulae to give people who complain loudly something to try.